cmsplugin_articles_ai/models/articles.py

```diff
@@ -106,6 +114,10 @@ class Article(models.Model):
         verbose_name=_("URL slug"),
         db_index=True,
     )
+    language = LanguageField(
```

Comment:
Sounds like a corner case. You could add another article lift plugin to lift the exceptional article.

Or, we could make this field nullable, but that just feels a bit confusing in the user's perspective.